brax/envs/fast.py

```diff
@@ -15,38 +15,68 @@
 # pylint:disable=g-multiple-import
 """Gotta go fast!  This trivial Env is for unit testing."""
 
-from brax import base
-from brax.envs.base import PipelineEnv, State
+from enum import Enum
+
 import jax
 from jax import numpy as jp
+from flax.core import FrozenDict
+
+from brax import base
+from brax.envs.base import PipelineEnv, State
+
+
+class ObservationMode(Enum):
```

Comment:
can you add a docstring? I had to scratch my head a bit to see what these modes meant